### PR TITLE
2443 remove aws security hub

### LINKED ISSUE #2443: Remove AWS Security Hub integration
## Explanation

Infection Monkey's mission has changed. The existing AWS Security Hub integration is not designed to work well with the new mission. It will be removed and rebuilt using the new infrastructure that is being built to support our new mission.

## Tasks
- [x] Remove the AWS Security Hub integration (0d) @ilija-lazoroski 
- [x] Review GitHub issues related to this report and close them as necessary (0d)

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Fixes #2443.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] ~Is the TravisCI build passing?~
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [x] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running Island locally and check if everything works.
* [x] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Review by cakekoa — APPROVED** [review]

File: docs/content/usage/integrations/aws-run-on-ec2-machine.md [hunk 1]
```diff
@@ -103,51 +102,3 @@ Here's the policy of the IAM role, as a JSON object:
     ]
 }
 ```
-
```
Comment (VakarisZ) [inline docs/content/usage/integrations/aws-run-on-ec2-machine.md:106]: Also remove `AWSSecurityHubFullAccess` permissions on line 30

File: monkey/monkey_island/cc/services/reporting/issue_processing/exploit_processing/exploiter_report_info.py [hunk 2]
```diff
@@ -1,14 +1,7 @@
 from dataclasses import dataclass
-from enum import Enum
 from typing import Union
 
 
-class CredentialType(Enum):
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/services/reporting/issue_processing/exploit_processing/exploiter_report_info.py:6]: Also unused in `monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js:49`

File: monkey/monkey_island/cc/services/reporting/report.py [hunk 3]
```diff
@@ -41,7 +41,6 @@
 )
 
 from .. import AWSService
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/services/reporting/report.py:43]: This is used for what?

**Review by VakarisZ — CHANGES_REQUESTED** [review]
Couple of small tweaks

File: monkey/monkey_island/cc/services/reporting/report.py (hunk 3 above)
Comment (ilija-lazoroski) [reply monkey/monkey_island/cc/services/reporting/report.py:43]: As it stands. Nowhere.

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**